.travis.yml

```diff
@@ -29,11 +29,8 @@ before_install:
   - nvm use 12
 install:
   - pip install -r requirements_test.txt
-  # if Travis is running Python 3, then
-  #   refresh the git submodule ./vendor/infogami
-  - if [ "$TRAVIS_PYTHON_VERSION" == "3.8" ]; then
-      pushd vendor/infogami && git pull origin master && popd;
-    fi
+  # refresh the git submodule ./vendor/infogami
+  - pushd vendor/infogami && git pull origin master && popd
```

Comment:
Let's add a new job for Py2 + Infogami master with `allow_failures`. Since we're actively developing on both infogami and openlibrary, we don't want everyone working on openlibrary to be blocked if a bug gets into infogami master.

(Since openlibrary depends on infogami, there's ~no chance that an openlibrary edit will cause infogami to break. Bugs can only be introduced in the other direction; i.e. an infogami edit can cause openlibrary to break (which is why openlibrary is pinned to specific version of infogami known to work). So the only thing we care about is if an infogami edit causes things to break.)